src/libs/actions/PersonalDetails.js

```diff
@@ -171,10 +171,14 @@ function getFromReportParticipants(reports) {
             Onyx.merge(ONYXKEYS.PERSONAL_DETAILS, formattedPersonalDetails);
 
             // The personalDetails of the participants contain their avatar images. Here we'll go over each
-            // report and based on the participants we'll link up their avatars to report icons.
+            // report and based on the participants we'll link up their avatars to report icons. This will
```

Comment:
Check the [Design Doc](https://docs.google.com/document/d/1xabico6VzVd_biN9qw2LFdLW6Lbr9wpN36etbHTeX8c/edit#heading=h.gy81859colv8) - we want to place this into a new method in `OptionsListUtils.js`

Reply:
OHH I didn't recognize the header for this part meant it should be a function there. Ok that makes sense I'll move it there.